datanode/gateway/graphql/new_market_resolver.go

```diff
@@ -96,3 +96,7 @@ func (r *newMarketResolver) RiskParameters(_ context.Context, obj *types.NewMark
 func (r *newMarketResolver) Metadata(_ context.Context, obj *types.NewMarket) ([]string, error) {
 	return obj.Changes.Metadata, nil
 }
+
+func (r *newMarketResolver) SuccessorConfiguration(ctx context.Context, obj *types.NewMarket) (*types.SuccessorConfiguration, error) {
+	return obj.Changes.Successor, nil
```

Comment:
same here, successor no longer exists

Reply:
No, this is the mapping for the protobuf SuccessorConfiguration:

```
// Configuration required to turn a new market proposal in to a successor market proposal.
message SuccessorConfiguration {
  // ID of the market that the successor should take over from.
  string parent_market_id = 1;
  // A decimal value between or equal to 0 and 1, specifying the fraction of the insurance pool balance that is carried over from the parent market to the successor.
  string insurance_pool_fraction = 2;
}
```